projects/ion/src/lib/popconfirm/component/popconfirm.component.ts

```diff
@@ -0,0 +1,43 @@
+import { CommonModule } from '@angular/common';
+import { Component, Input } from '@angular/core';
+import { Subject } from 'rxjs';
+import { IonAlertComponent } from '../../alert';
+import { IonButtonComponent } from '../../button';
+import { IonDividerComponent } from '../../divider';
+import { IonPopConfirmProps } from '../types';
+
+@Component({
+  standalone: true,
+  selector: 'ion-popconfirm',
+  imports: [
+    CommonModule,
+    IonAlertComponent,
+    IonDividerComponent,
+    IonButtonComponent,
+  ],
+  templateUrl: './popconfirm.component.html',
+  styleUrls: ['./popconfirm.component.scss'],
+})
+export class IonPopConfirmComponent {
+  @Input() ionPopConfirmTitle!: IonPopConfirmProps['ionPopConfirmTitle'];
```

Comment:
Migrar para signal inputs

```suggestion
  ionPopConfirmTitle! = input<IonPopConfirmProps['ionPopConfirmTitle']>();
```